### PR TITLE
metalbear-cloud login

### PR DESCRIPTION
**Pull request by DmitryDodzin**
Add login feature to mirrord-cli

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+### Added
+- mirrord-cli add login command for loggin in to metalbear-cloud
```
Comment (aviramha) [inline CHANGELOG.md:11]: typo `logging`

File: mirrord-auth/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    #[error(transparent)]
+    ConfigParseError(#[from] serde_json::Error),
+    #[error(transparent)]
+    #[cfg(feature = "webbrowser")]
+    ConfigRequestError(#[from] reqwest::Error),
+}
+
+type Result<T> = std::result::Result<T, AuthenticationError>;
+
+impl AuthConfig {
```
Comment (aviramha) [inline mirrord-auth/src/lib.rs:48]: Using of a raw file is quite "dangerous" - I know that many apps do it, but if it's possible to use a crate such as this https://github.com/hwchen/keyring-rs then let's do it (if not, open an issue to later transition, but would be easier to start with such solution)

**Review by aviramha — CHANGES_REQUESTED** [review]

File: mirrord-auth/src/lib.rs (hunk 2 above)
Comment (DmitryDodzin) [reply mirrord-auth/src/lib.rs:48]: the 'username' for the keyring should be the OS username or something from cli input (with default value)?

Comment (aviramha) [reply mirrord-auth/src/lib.rs:48]: os username sounds good

Comment (aviramha) [reply mirrord-auth/src/lib.rs:48]: (in the example they use `whoami`)

**Review by aviramha — APPROVED** [review]

**Pull request merged**